src/main/java/seedu/duke/task/Book.java

```diff
@@ -95,8 +95,8 @@ public String toStringBook(boolean isList) {
         return returnString;
     }
 
-
-    public String toFileBook() {
+    @Override
+    public String toFile() {
         String isDoneString = (isDone) ? "1" : "0";
```

Comment:
This part, could you change it to `isReturnString = (isReturn) ? "1" : "0"; `

Reply:
Yea sure